configure.ac

```diff
@@ -88,6 +88,12 @@ AC_ARG_ENABLE([mining],
   [enable_mining=$enableval],
   [enable_mining=yes])
 
+AC_ARG_ENABLE([proton],
+  [AS_HELP_STRING([--disable-proton],
```

Comment:
Yes, I was thinking about this.  I made the decision that since this is a build option it should reflect the name of the third party library to make it obvious what is being linked in.  Also this matches other options such as -with-miniupnpc, whereas options like --enable-wallet and --enable-mining reflect a native feature of the application.  We can always change this later since most users will not build by setting configure options but instead use build.sh, where we could add options such as --disable-(zmq|amqp)-notifications.